lomap/graphgen.py

```diff
@@ -332,12 +362,16 @@ def minimizeEdges(self):
             if len(subgraph.edges()) > 2:   # Graphs must have at least 3 edges to be minimzed
 
                 for edge in weightsList:
-
-                    subgraph.remove_edge(edge[0], edge[1])
-
-                    if self.checkConstraints(subgraph, numberOfComponents) == False:
-                        subgraph.add_edge(edge[0], edge[1], similarity = edge[2], strict_flag = True)
-
+                    #sliu 07/17 add radial option here to avoid removing edges connect to the lead compound
```

Comment:
Changelog not necessary, though this may be a good place to have a comment saying what you are doing! The current comment is incomprehensible to me.